.ci/jenkins/build-int-test/Jenkinsfile

```diff
@@ -6,12 +6,12 @@ pipeline {
         DOCKER_IMAGE_PREFIX = 'opera_pge'
         DOCKER_TAG = """${sh(
                      returnStdout: true,
-                     script: 'echo ${GIT_BRANCH##*/}'
+                     script: 'echo ${GIT_BRANCH##*/}-build-int-test-$$'
```

Comment:
I ran into a problem where an automated "OPERA PGE Build/Test Pipeline" build called .ci/scripts/cleanup.sh for the same git branch that I had manually started a "OPERA PGE Build/Integration Test Pipeline".
The Build/Test pipeline removed 2 of the images from the Build/Integration pipeline (because the image tags were the same for both pipelines), and both pipelines failed.
The process ID ($$) is sufficient to avoid pipeline cleanup.sh scripts from removing other pipeline images, but I thought adding the Jenkinsfile path might help with troubleshooting going forward.